### PR TITLE
Add remaining keras layer mappings

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

cropping2d 3d
spatialdropout1d 2d 3d
locallyconnected2d

## How was this patch tested?

Jenkins

### CODE REVIEW

File: pyspark/bigdl/keras/converter.py [hunk 1]
```diff
@@ -816,6 +816,31 @@ def create_cropping1d(self):
...
+        bigdl_order = self.get_bdim_order()
+        blayer = BLayer.Cropping2D(heightCrop=self.klayer.cropping[0],
+                                   widthCrop=self.klayer.cropping[1],
+                                   data_format=bigdl_order)
+        return blayer
+
+    def create_cropping3d(self):
+        if "dim_ordering" not in self.config:
+            warnings.warn("Cannot find dim_ordering from json definition. Using the default instead.")
+        if self.klayer.dim_ordering == "th":
```
Comment (zhichao-li) [inline pyspark/bigdl/keras/converter.py]: we should use a existing mapping method  for this.

Comment (hkvision) [reply pyspark/bigdl/keras/converter.py]: added a 3d mapping

**Comment by zhichao-li** [comment]
LGTM

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/PR-V/job/BigDL-PR-V/592/

**Pull request merged**